haskell/src/Ligo/BaseDAO/ErrorCodes.hs

```diff
@@ -96,8 +96,8 @@ notDelegate :: Natural
 notDelegate = 120
 
```

Comment:
This commit (bce9d640b063f475d64448746320f7948bb5937b) has no description in its message (even tho it will be a pretty simple one).

Reply:
Have expanded the commit message.